### DISCUSSION TITLE
Xiaomi Robot Vacuum Mop 2S

### DISCUSSION THREAD

**Discussion by rucanunes** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#510] [General]
Hello,

Is this integration compatible with the Xiaomi Robot Vacuum Mop 2S?

Thank you!

**Comment by PiotrMachowski** [comment]
This is not an integration, just a lovelace card - you need to install an integration that will support your vacuum to make it work.

**Comment by rucanunes** [comment]
After some search I didn't found a specific integration for this model.
Could someone please tell me if there is one?
